docs/ref/contrib/gis/install/geolibs.txt

```diff
@@ -219,27 +220,25 @@ First download the latest GDAL release version and untar the archive::
 
     $ wget https://download.osgeo.org/gdal/X.Y.Z/gdal-X.Y.Z.tar.gz
     $ tar xzf gdal-X.Y.Z.tar.gz
-    $ cd gdal-X.Y.Z
 
-Configure, make and install::
+For GDAL 3.6.x and greater, releases only support builds using ``CMake``. To
```

Comment:
See [release notes](https://github.com/OSGeo/gdal/blob/v3.6.0/NEWS.md#in-a-nutshell): _"CMake is the only build system available in-tree. autoconf and nmake build systems have been removed"_.